rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -405,7 +462,7 @@ s32 cellCameraIsOpen(s32 dev_num)
 
 	const auto g_camera = fxm::get<camera_thread>();
 
-	bool is_open = g_camera->is_open;
+	const bool is_open = g_camera->is_open;
 	return is_open;
```

Comment:
why not just return g_camera->is_open;

Reply:
I think these are from when I had the lock scopes and they stuck around after neko told me to remove the scopes.